docs/content/contribute.md

```diff
@@ -86,14 +88,17 @@ reach out to a maintainer on [Slack][slack].
 [skeletor]: https://github.com/deislabs/porter-skeletor
 [mixin-dev-guide]: /mixin-dev-guide/
 [roadmap]: /roadmap
+[existing-mixins]: https://github.com/deislabs/porter-packages/blob/main/mixins/index.json
+[mixin-ideas]: https://github.com/deislabs/porter/issues?q=is%3Aissue+is%3Aopen+label%3A%22mixin+idea%22
+[twitter]: https://github.com/
```

Comment:
This twitter link looks like it's dangling.

Reply:
oops! thanks yes it's not used anywhere